kwok/charts/crds/karpenter.sh_nodeclaims.yaml

```diff
@@ -32,6 +32,10 @@ spec:
         - jsonPath: .status.conditions[?(@.type=="Ready")].status
           name: Ready
           type: string
+        - jsonPath: .status.conditions[?(@.type=="Drifted")].status
```

Comment:
I think we need to move the wide output after all of the non-wide output so things don't shift around in a weird way

Reply:
Also, given that this could be blank, does it make more sense for this to be at the end after all of the non-blank output